api/db/changelog/db.changelog-218-drop-unused-columns-user-table.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="thibault" id="db.changelog-218-drop-unused-columns-user-table">
+    <dropColumn columnName="compliance_training_bypass_time" tableName="user"/>
```

Comment:
All bypass and completion fields are now handled by the `user_access_module` table.